src/test/java/be/syntra/devshop/DevshopFront/testutils/ProductUtils.java

```diff
@@ -101,7 +98,6 @@ public static Product getOtherDummyNonArchivedProduct() {
                 .description("another description")
```

Comment:
remove getOtherDummyNonArchivedProduct(). (Found the missing 0.99) 😄

Reply:
removed, i'm glad you found it :smile_cat: :smile: